src/less/widgets/widgets-general.less

```diff
@@ -21,7 +21,7 @@ a {
 }
 
 .pf-box-shadow {
-  box-shadow: 0 0 40px rgba(0,0,0,.5);
+  box-shadow: 0 4px 4px -4px rgba(0,0,0,.5);
```

Comment:
I like the new shadow! Though I noticed on left or right aligned widgets it goes missing.
<img width="325" alt="screen shot 2016-02-10 at 11 14 41 am" src="https://cloud.githubusercontent.com/assets/5973633/12958740/07f997f4-cfe8-11e5-85a8-67bc72825026.png">
Probably not a big deal, but it would be nice to adjust the shadow for left or right positioned widgets. (This is in Themes example.html, for reference).